src/bindings/Promise2.resi

```diff
@@ -1,236 +0,0 @@
-// The +'a marks the abstract type parameter as covariant, which essentially means that
```

Comment:
Promise2 is now gone, since its official code has been moved into `rescript-core`.